src/MainWindow.vala

```diff
@@ -151,11 +162,22 @@ public class AppCenter.MainWindow : Gtk.ApplicationWindow {
         installed_view = new Views.InstalledView ();
         search_view = new Views.SearchView ();
 
+        network_alert_view = new Granite.Widgets.AlertView (_("Network is Not Available"),
```

Comment:
Ah yeah I'm not familiar with the rules of title case as we don't do that in the Dutch language. Fixed it in my last commit :)